apps/grader-host/package.json

```diff
@@ -19,13 +21,12 @@
     "ajv": "^8.12.0",
     "async": "^3.2.4",
     "async-stacktrace": "^0.0.2",
-    "aws-sdk": "^2.1369.0",
     "byline": "^5.0.0",
     "chai": "^4.3.7",
     "dockerode": "^3.3.5",
     "fs-extra": "^11.1.1",
     "logform": "^2.5.1",
-    "s3-streamlogger": "1.7.0",
+    "s3-streamlogger": "^1.9.0",
```

Comment:
We previously pinned this to an earlier version because the internal v3 upgrade broke some things for us. However, now that we *want* the v3 SDK, I unpinned this and updated our usage to explicitly pass in a `region`.